autoinit/initializer/initialize_weights.py

```diff
@@ -46,18 +46,28 @@ class AutoInit:
 
     def __init__(self,
                  weight_init_config: Dict = None,
+                 custom_distribution_estimators: Dict = None,
```

Comment:
Best to put new args at the end of the list for backwards compatibility
just in case ordered args are called *without* variable names "foo=bar" in the list.

Reply:
done